src/main/java/seedu/task/logic/parser/UncompleteCommandParser.java

```diff
@@ -0,0 +1,25 @@
+package seedu.task.logic.parser;
+
+import static seedu.task.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Optional;
+
+import seedu.task.logic.commands.Command;
+import seedu.task.logic.commands.IncorrectCommand;
+import seedu.task.logic.commands.UncompleteCommand;
+
+public class UncompleteCommandParser {
```

Comment:
This class seems rather redundant. Could we not have put the logic into CompleteCommandParser?